vetiver/vetiver_model.py

```diff
@@ -44,7 +44,7 @@ def __init__(
         metadata: dict = None,
         **kwargs
     ):
-        translator = create_translator(model, ptype_data)
+        translator = create_handler(model, ptype_data)
```

Comment:
I wonder if a move that would take pressure off of getting the handlers right inside vetiver might be allowing users to pass either a model to pass to create_handler, or a handler itself. This way, if someone needs to fix / customize something about handling, they can define their own. (e.g. if the scikit or torch handlers don't do what they need).